charts/crljanitor/templates/deployment.yaml

```
@@ -71,6 +71,15 @@ spec:
             secretKeyRef:
               name: crl-janitor-track-resource-pubsub
               key: subscription
+        # Environment variables for OpenCensus resource auto detection.
+        - name: CONTAINER_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.labels['app']
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
```

```suggestion
        - name: NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
        - name: VERSION
          value: {{ .Values.appVersion }}
```
It may be worth adding something like this so that an openCensus tag with the application version can be added to any metrics. I'm not sure if  Opencensus supports automatically extracting this value as with the above, but this is a common pattern when collecting metric data.

---

Discussed offline. It appears there is a system_labels.container_image field in stackdriver that is collected however it contains a full gcr path so may be problematic to work with. We are going to try the existing container image field first and may revisit this if that proves unmanageable.